redirects.yaml

```diff
@@ -395,84 +395,116 @@ redirects:
   - source: /debug-web-vitals-in-the-field/
     destination: /debug-performance-in-the-field/
 
-   # Redirects for Lighthouse accessibility audits, now deferring to axe docs.
-  - source: /accesskeys/
+  # Redirects for Lighthouse accessibility content.
+  - source: /lighthouse-accessibility
+    destination: https://developer.chrome.com/docs/lighthouse/accessibility/
+  - source: /accessibility-scoring
+    destination: https://developer.chrome.com/docs/lighthouse/accessibility/scoring/
+  # Redirects accessibility audits, now deferring to axe docs.
+  - source: /accesskeys
     destination: https://dequeuniversity.com/rules/axe/4.4/accesskeys
-  - source: /bypass/
+  - source: /bypass
```

Comment:
I realized we are 404ing for anyone visiting without a trailing slash (e.g. https://web.dev/bypass currently 404s, though https://web.dev/bypass/ correctly redirects). Dropping the trailing slash handles both cases.